File: scratch/find_combined.js
const fs = require('fs');
const path = require('path');

const targets = ['submission', 'docs', 'experiments', 'answer_key', 'artifacts', 'scripts'];

function walk(dir) {
  let results = [];
  try {
    const list = fs.readdirSync(dir, { withFileTypes: true });
    for (const ent of list) {
      const full = path.join(dir, ent.name);
      if (ent.isDirectory()) {
        results.push(...walk(full));
      } else if (ent.isFile() && /\.(json|md|py|tex|txt)$/.test(ent.name)) {
        try {
          const content = fs.readFileSync(full, 'utf8');
          if (content.includes('0.6486') || content.includes('0.6073') || content.includes('Combined-37')) {
            console.log(`Found in ${full}`);
          }
        } catch (e) {}
      }
    }
  } catch (e) {}
  return results;
}

targets.forEach(t => walk(t));

File: scratch/find_combined.py
import os

def search_files():
    targets = ['submission', 'docs', 'experiments', 'answer_key', 'artifacts', 'scripts']
    for t in targets:
        for root, dirs, files in os.walk(t):
            for f in files:
                if f.endswith(('.json', '.md', '.py', '.tex', '.txt')):
                    p = os.path.join(root, f)
                    try:
                        with open(p, 'r', encoding='utf-8', errors='ignore') as fp:
                            txt = fp.read()
                            if '0.6486' in txt or '0.6073' in txt or 'Combined-37' in txt:
                                print(f"Found in {p}")
                    except Exception as e:
                        pass

if __name__ == '__main__':
    search_files()

